picard/ui/edittagdialog.py

```diff
@@ -138,7 +138,11 @@ def tag_selected(self, index):
 
     def edit_value(self):
         item = self.value_list.currentItem()
-        if item and not self.value_list.isPersistentEditorOpen(item):
+        if item:
+            # Do not initialize editing if editor is already active. Avoids flickering of the edit field
+            # when already in edit mode. `isPersistentEditorOpen` is only supported in Qt 5.10 and later.
+            if hasattr(self.value_list, 'isPersistentEditorOpen') and self.value_list.isPersistentEditorOpen(item):
+                return
```

Comment:
Just in case this would come up: I intentionally put this into a separate `if`. I think it makes it easier to comment this workaround and read what's going on.